src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,6 +14,8 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
+    private static int nextSequenceNumber;
```

Comment:
nextSequenceNumber should be initiated to 1. This is because your first Person object created would have index 0 instead of 1.